tests/system/aiplatform/test_e2e_tabular.py

```diff
@@ -106,7 +109,7 @@ def test_end_to_end_tabular(self, shared_state):
             optimization_objective="minimize-rmse",
         )
 
-        # Kick off both training jobs, AutoML job will take approx one hour to run
+        # Kick off both training jobs to check they are started correctly, then cancel the AutoML job
```

Comment:
Hey Sara, small syntax nit, but can we remove the line space between the comment and the code for the comments that have that space?

Reply:
Done